wordpress-mobile/WordPress-iOS
# Issue/9670 Universal Links: Reader

Implements #9670. This PR follows on from #9702, and adds a universal links route for Reader:

* `/read`
* `/discover`
* `/read/search`
* `/read/a8c`
* `/activities/likes`
* `/following/manage`
* `/read/list/:username/:list_name`
* `/tag/:tag_name`
* `/read/feeds/:feed_id/posts/:post_id`
* `/read/blogs/:blog_id/posts/:post_id`

There is a separate small WordPressKit-iOS PR to go along with this one: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/23

**To test:**

* Same as the previous PRs, add an `applinks:` entry to the Associated Domains capability for the test domain I provided.
* In Safari on your device, open the test page I created and try each of the Reader link. Check that the various items in the Reader section of the app launch as expected. Please let me know if you have any questions!

Thanks you!!
